docker/base-deps/Dockerfile

```diff
@@ -121,7 +122,7 @@ PIP_PKGS=(
 )
 
 # Install uv
-wget -qO- https://astral.sh/uv/install.sh | sudo env UV_UNMANAGED_INSTALL="/usr/local/bin" sh
+wget -qO- https://astral.sh/uv/install.sh | sudo -E env UV_UNMANAGED_INSTALL="/usr/local/bin" sh
```

Comment:
We may need proxy to download files in China Mainland and need to pass proxy env to sudo. So I add "-E" here.